devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -435,6 +436,19 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         }
     }
 
+     if (!(config.check("CalibrationConfig") && config.find("CalibrationConfig").isList())) {
+        yInfo() << LogPrefix << "CalibrationConfig option not found or not valid, all the model joints are selected.";
+        pImpl->CalibrationConfig.clear();
```

Comment:
In order to maintain backcompatibility, if the option is not found, the zero configuration should be used. Here you are setting an empty `std::vector` that would cause seg fault when called.

You should instead initialize and std:::vector of zeros with the same size as the joint

Reply:
I have a negative number of ideas on how to do it.

Reply:
Solved in https://github.com/robotology/human-dynamics-estimation/pull/341/commits/e59e7907e431dbdd0950bfd52ac89509d1abe1f0 and https://github.com/robotology/human-dynamics-estimation/pull/341/commits/c61dd9e59cbb2ef66a037f55b7b8b85e9d83634a